### PR TITLE
Issue #140: Made Header Responsive

### PR DESCRIPTION
**Pull request by NishidhJain**
This will close issue #140

@max-programming Please have a look at it. Thank You!

### CODE REVIEW

**Comment by max-programming** [comment]
Thank you very much!

@all-contributors please add @NishidhJain for design, code

**Comment by max-programming** [comment]
Can you fix the conflicts so I can merge?

**Comment by NishidhJain** [comment]
Sure, I am working on it @max-programming

**Comment by NishidhJain** [comment]
Hey @max-programming, I have resolved the conflicts and its ready to merge. Please have a look at it.

Also please have a look at this PR https://github.com/max-programming/hacktoberfest-projects/pull/136

One more request, can you please consider this contribution under hacktoberfest and add the label for the same.

Thank You!!

**Comment by max-programming** [comment]
@NishidhJain I'll be reviewing the other PR later. And also don't worry once the PR gets merged, it will be counted in Hacktoberfest

**Comment by max-programming** [comment]
@all-contributors please add @NishidhJain for design, code

**Comment by prithvi2k2** [comment]
@max-programming @NishidhJain GitHub logo is broken, i mean it is shown in black color instead of white... I suspect this PR commited that change

File: components/Header.tsx [hunk 1]
```diff
@@ -5,40 +5,51 @@ import { useRouter } from 'next/router';
...
+              <BsPeopleFill size="1.5rem" color='white' title="Contributors" />
+            </Link>
+
+            <a
+              href="https://github.com/max-programming/hacktoberfest-projects"
+              target="_blank"
+              rel="noreferrer"
+              className="btn btn-square btn-ghost umami--click--github-button"
+            >
+              <IoLogoGithub size="1.5rem" title="GitHub" />
```
Comment (prithvi2k2) [inline components/Header.tsx:45]: Add `color='white'` to `IoLogoGithub`

Comment (NishidhJain) [reply components/Header.tsx:45]: image

@prithvi2k2  I can see its already white, can you share the screenshot how its looking for you?

Comment (prithvi2k2) [reply components/Header.tsx:45]: That is a different deployment by hacktoberfest, check out original deployment @ https://finder.usmans.me/

Comment (prithvi2k2) [reply components/Header.tsx:45]: Wait, even that is gone now 
image

Comment (NishidhJain) [reply components/Header.tsx:45]: image

Its already white on it as well

Comment (prithvi2k2) [reply components/Header.tsx:45]: @NishidhJain Maybe it's your browser messing up with colors, try it in dark mode or mobile - preferably incognito

Comment (prithvi2k2) [reply components/Header.tsx:45]: > image
> 
> Its already white on it as well

I tried with different browsers and on mobile too, did you check it with other browsers or platforms? It's still black for me
image

Comment (NishidhJain) [reply components/Header.tsx:45]: I just checked in incognito mode and on mobile device its showing in white. Also, I have dark mode turned on. @max-programming Can you please confirm how its looking on your device?

File: components/Header.tsx [hunk 2]
```diff
@@ -5,40 +5,51 @@ import { useRouter } from 'next/router';
...
+          <Link href="/">
+            <img src="/hacktoberfest.svg" alt="Hacktoberfest" className="h-12 w-auto sm:h-auto" />
           </Link>
-          <a
-            href="https://github.com/max-programming/hacktoberfest-projects"
-            target="_blank"
-            rel="noreferrer"
-            className="btn btn-square btn-ghost umami--click--github-button"
-          >
-            <IoLogoGithub size="1.5rem" color='white' title="GitHub" />
```
Comment (prithvi2k2) [inline components/Header.tsx:37]: It actually had that attribute before, maybe you removed it as were not seeing any effect with or without this in your browser environment

File: components/Header.tsx (hunk 1 above)
Comment (prithvi2k2) [reply components/Header.tsx:45]: > I just checked in incognito mode and on mobile device its showing in white. Also, I have dark mode turned on. @max-programming Can you please confirm how its looking on your device?

oh well. that is the cause of issue then, dark mode - as far as I've seen, it messed up many elements for me many times on various pages... I'm talking about 3rd party extensions for dark mode and Forced Dark Modes in browser. Can you turn it off and check?

File: components/Header.tsx (hunk 2 above)
Comment (NishidhJain) [reply components/Header.tsx:37]: I would add it back, give me sometime, but it seems like its not making any difference

Comment (prithvi2k2) [reply components/Header.tsx:37]: It is, I've tested on 3 different devices now - without dark mode, just the normal mode. It surely wasn't black yesterday and day-before-yesterday

Comment (prithvi2k2) [reply components/Header.tsx:37]: > I would add it back, give me sometime, but it seems like its not making any difference

Turn off dark mode and try again on your devices

Comment (NishidhJain) [reply components/Header.tsx:37]: Got the issue, its happening in light mode, its working fine on dark mode. Will fix this. @prithvi2k2 Thanks for highlighting.

Comment (prithvi2k2) [reply components/Header.tsx:37]: Yes, Afaik the design language of this site is based on DARK coloring... so, the goal might be to run it in dark mode only even though client is on light or dark mode. So, defining static color to logo, irrespective of light/dark modes is the solution.

Comment (NishidhJain) [reply components/Header.tsx:37]: https://github.com/max-programming/hacktoberfest-projects/pull/149/commits/b95ae80f617af25c75615c169f5e6dc6f4c00e2e

Added back color attribute in above commit @prithvi2k2

**Pull request merged**